src/test/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCountIntegrationTest.java

```diff
@@ -524,9 +524,10 @@ private void compareTableFiles(File outputFile, File expectedOutputFile, final S
         private String[] columnNames;
 
         public static Table fromFile(final File file) throws IOException {
-            final FileReader fr = new FileReader(file);
-            final Table result = new Table(fr);
-            fr.close();
+            Table result = null;
+            try (final FileReader fr = new FileReader(file)) {
+                result = new Table(fr);
```

Comment:
Can you just say `return new Table(fr)` inside of the try block? Or will the compiler complain?